lib/console_yasttest.pm

```diff
@@ -5,20 +5,28 @@ use base 'y2logsstep';
 use strict;
 use testapi;
 
-sub assert_service_widget {
+sub change_service_configuration {
     my ($self, %args) = @_;
-    my $after_writing_conf = $args{after_writing_conf} || 'alt-f';
-    my $after_reboot       = $args{after_reboot}       || 'alt-a';
+    my $after_writing_ref = $args{after_writing};
```

Comment:
before creating this variable had sense because in case nothing passed into a function it will assign default value. but you remove default values so you can : 

```
sub change_service_configuration {
 my ($self, %args) = @_;
 change_service_configuration_step('after_writing_conf', $after_writing_ref) if $args{after_writing};
 change_service_configuration_step('after_reboot',       $after_reboot_ref)  if $args{after_reboot};
}
```
but you can say that it is matter of taste and I will not argue :)

Reply:
There were many combinations and the default values didn't really help. For clarity is all passed to the subroutine and if not, means that we want to select specific field to change, as you said, a matter of taste :)